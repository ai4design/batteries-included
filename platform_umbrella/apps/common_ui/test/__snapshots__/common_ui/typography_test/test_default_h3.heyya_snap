<h3 class="text-xl font-medium text-gray-600 dark:text-gray-200 ">
  Test H3 Header
</h3>