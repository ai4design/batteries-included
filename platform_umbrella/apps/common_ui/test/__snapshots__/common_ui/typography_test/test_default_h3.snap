"<h3 class=\"text-lg sm:text-2xl text-bold my-2\">\n  Test H3 Header\n</h3>"